mmseg/datasets/transforms/transforms.py

```diff
@@ -1226,3 +1226,201 @@ def __repr__(self):
         repr_str += f'edge_width={self.edge_width}, '
         repr_str += f'ignore_index={self.ignore_index})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class BioPatchCrop(BaseTransform):
+    """Crop the input patch for medical image & seg.
+
+    Required Keys:
+
+        - img
+        - gt_seg_map
+
+    Modified Keys:
+
+        - img
+        - img_shape
+        - gt_seg_map
+
+
+    Args:
+        crop_size (Union[int, Tuple[int, int, int]]):  Expected size after
+            cropping with the format of (z, y, x). If set to an integer,
+            then cropping width and height are equal to this integer.
+        force_fg (bool): Cropped patch must contain foreground.
+        crop_mode (str): The crop mode for biomedical patch cropping.
+            Default: nnUNet
```

Comment:
```suggestion
            Default: nnUNet
```
I suggest to provide `nnUNet` and `random` two options on `crop_mode`, other strings are illegal.